MessengerOnFileForLinux/Common/src/TimeManagment.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <chrono>
+#include <string>
+
+namespace TimeManagment
+{
+    const time_t getActualDate();
```

Comment:
look, when you like to do auto a = 7. but you dislike auto here? Think about it.